pkg/reconciler/testing/listers.go

```diff
@@ -22,15 +22,15 @@ import (
 	fakekubeclientset "k8s.io/client-go/kubernetes/fake"
 	corev1listers "k8s.io/client-go/listers/core/v1"
 	"k8s.io/client-go/tools/cache"
-	gwv1alpha1 "sigs.k8s.io/gateway-api/apis/v1alpha1"
+	gwv1alpha2 "sigs.k8s.io/gateway-api/apis/v1alpha2"
 
 	networking "knative.dev/networking/pkg/apis/networking/v1alpha1"
 	fakeservingclientset "knative.dev/networking/pkg/client/clientset/versioned/fake"
 	networkinglisters "knative.dev/networking/pkg/client/listers/networking/v1alpha1"
 	"knative.dev/pkg/reconciler/testing"
 
 	fakegatewayapiclientset "knative.dev/net-gateway-api/pkg/client/gatewayapi/clientset/versioned/fake"
-	gwlisters "knative.dev/net-gateway-api/pkg/client/gatewayapi/listers/apis/v1alpha1"
+	gwlisters "knative.dev/net-gateway-api/pkg/client/gatewayapi/listers/apis/v1alpha2"
```

Comment:
This might be a nit: I'd prefer this alias be named `gatewayListers`.  Looking at the other aliases they are spelled out, and also it'd be more inline with `gatewayv1alpha2`.